packages/app-cli/app/command-done.test.ts

```diff
@@ -13,7 +13,7 @@ describe('command-done', () => {
 	});
 
 	it('should make a note as "done"', async () => {
-		const note = await Note.save({ title: 'hello', is_todo: 1, todo_completed: 0 });
+		const note = await Note.save({ title: 'hello', is_todo: 1, todo_completed: 0, parent_id: '' });
```

Comment:
The above change fixes a test failure. This pull request added a Redux initialization step to the CLI test setup. As a result, additional processing is done when saving a note. This includes a check for the presence of parent_id (possibly [this check](https://github.com/laurent22/joplin/blob/f6fef5a8ec489695bda1cc6d6bffe35625ef608f/packages/lib/services/trash/index.ts#L21)).